src/vdxf/scopes.ts

```diff
@@ -16,4 +16,34 @@ export const IDENTITY_AGREEMENT: VDXFKeyInterface = {
     namespace: "i5w5MuNik5NtLcYmNzcvaoixooEebB6MGV",
     name: "vrsc::identity.authentication.permission.agree"
   }
-};
\ No newline at end of file
+};
+
+export const ATTESTATION_READ_REQUEST: VDXFKeyInterface = {
+  vdxfid: "iNqLaiDJjcADGCvXcQZnPqwTqMXzQbDCFu",
+  hash160result: "fbd8fe825062b19a9bd26dc51b6f768828115ad4",
+  qualifiedname: {
+    namespace: "i5w5MuNik5NtLcYmNzcvaoixooEebB6MGV",
+    name: "vrsc::identity.attestation.read"
+  }
+};
+
+export const PROFILE_DATA_READ_REQUEST: VDXFKeyInterface = {
+  vdxfid: "iFYznrRxyHaoJZ91cFDSYGT4szETf4RLRV",
+  indexid: "xLP7Fes3pboTvj23TvsbWeybueFUatbQQh",
+  hash160result: "d3f2168aad438c6c6eab8f3384458cbebf027a84",
+  qualifiedname: {
+    namespace: "i5w5MuNik5NtLcYmNzcvaoixooEebB6MGV",
+    name: "vrsc::profile.data.view"
```

Comment:
Should this have the identity prefix? Idk for sure, we can discuss

Reply:
To discuss maybe 

`vrsc::identity.profile.proposed.data.view`
not sure, your basically asking if you can see your PII data that's in the mobile app